engine/schema/src/main/java/com/cloud/user/UserAccountVO.java

```diff
@@ -131,12 +132,6 @@ public enum Setup2FAstatus {
     public UserAccountVO() {
     }
 
-    @Override
-    public String toString() {
-        return String.format("UserAccount %s.", ReflectionToStringBuilderUtils.reflectOnlySelectedFields
-                (this, "id", "uuid", "username", "accountName"));
-    }
-
```

Comment:
Not sure if we have to retain this: https://github.com/apache/cloudstack/pull/9873/files#diff-ad04db4978745b82495c1971e127ef6d1a89fd05ff28530b775bd5ae0c0a848fR134-R138